package.json

```diff
@@ -94,7 +94,7 @@
       "dot-location": [2, "property"],
       "no-implied-eval": 2,
       "no-loop-func": 2,
-      "no-magic-numbers": 1,
+      "no-magic-numbers": [1, {"ignore": [-1, 0, 1, 2]}],
```

Comment:
I feel like for tuples we should rather use destructuring than indexes. For strings, magic numbers are fine, so this looks okay to me.